extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -122,4 +123,20 @@
      */
     @ConfigItem(defaultValue = "false")
     public boolean alwaysCacheBaseImage;
+
+    /**
+     * List of target platforms. Each platform is defined using the pattern: \<os\>|\<arch\>[/variant]|\<os\>/\<arch\>[/variant]
+     * ex: linux/amd64,linux/arm64/v8. If not specified, OS default is linux and architecture default is amd64
+     * 
+     * If more than one platform is configured, it is important to note that the base image has to be a Docker manifest or an
+     * OCI image index containing a version of each chosen platform
+     * 
+     * It doesn't work with native images, as cross-compilation is not supported
+     * 
+     * Target Platform is a incubating feature of Jib. See <a href=
+     * "https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image">Jib
+     * FAQ</a> for more information
+     */
+    @ConfigItem
+    public Optional<Set<String>> platforms;
```

Comment:
I think it can be done in a follow-up PR but I wonder if having a Converter here wouldn't be useful.

That would allow to validate the config and throw errors early.

There is an example here: https://github.com/quarkusio/quarkus/blob/master/core/runtime/src/main/java/io/quarkus/runtime/configuration/LocaleConverter.java .

Note that it's just a suggestion, maybe it wouldn't work. And I think we could merge this as is anyway and improve later.

Reply:
Sounds like a good idea, thanks!